### ISSUE TITLE
Less indenting in layers panel

### ISSUE THREAD

**Issue by Kilian** [modulz/prototype-feedback#6]
The indenting in the layers panel is pretty large and seems to increase by 30px for every level, which effectively means anything beyond 5 levels deep, or with a particularly long title, is lost. Why so much? 16px indenting looks great:

![Selection_661](https://user-images.githubusercontent.com/41970/102890728-3b272300-445d-11eb-8f74-ee583273d43f.png)

**Comment by StephenHaney** [comment]
Definitely something we need to address.

**Comment by randito** [comment]
Agreed.  It's hard to see what's happening.  Is there a way to make it wider too?  I want to adjust the width of that pane and make it wider and more usable, especially for deeply nested elements.

**Comment by naher94** [comment]
Adjustable panel width would be great!
